platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/SyncManagerTest.java

```diff
@@ -150,10 +150,7 @@ public EventCreationRuleResponse shouldCreateEvent(BaseEvent selfParent, BaseEve
 
     @BeforeAll
     static void beforeAll() {
-        new TestConfigBuilder()
-                .withValue("sync.maxIncomingSyncsInc", 10)
-                .withValue("sync.maxOutgoingSyncs", 10)
-                .getOrCreateConfig();
+        new TestConfigBuilder().getOrCreateConfig();
```

Comment:
Is this necessary if not setting any values?

Reply:
Nope, removed